azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,298 @@ def load_iothub_help():
+        "iot hub message-endpoint create storage-container"
+    ] = """
+        type: command
+        short-summary: Add a Storage Container endpoint for an IoT Hub.
+        examples:
+          - name: Create a key-based Storage Container endpoint for an IoT Hub.
+            text: >
+                az iot hub message-endpoint create storage-container -n {iothub_name} --en {endpoint_name} --container {container_name} --endpoint-account {account_name}
+          - name: Create a Storage Container endpoint for an IoT Hub using a connection string. The endpoint uri is omitted.
+            text: >
+                az iot hub message-endpoint create storage-container -n {iothub_name} --en {endpoint_name} -c {connection_string} --container {container_name}
+          - name: Create a Storage Container endpoint for an IoT Hub using system assigned identity with the given batch frequency, chunk size, and file name format. The endpoint must be specified.
+            text: >
+                az iot hub message-endpoint create storage-container -n {iothub_name} --en {endpoint_name} --endpoint-uri {endpoint_uri} --auth-type identityBased --container {container_name} -b {batch_frequency} -w {chunk_size} --ff {file_format}
+          - name: Create a Storage Container endpoint for an IoT Hub using user assigned identity with json encoding. The endpoint must be specified.
+            text: >
+                az iot hub message-endpoint create storage-container -n {iothub_name} --en {endpoint_name} --endpoint-uri {endpoint_uri} --auth-type identityBased --container {container_name} --encoding json
+    """
+
+    helps[
+        "iot hub message-endpoint list"
+    ] = """
+        type: command
+        short-summary: Get information on all the endpoints for an IoT Hub.
+        long-summary: Get information on all endpoints in an IoT Hub. You can also specify which endpoint type you want to get information on.
+        examples:
+          - name: Get all the endpoints from an IoT Hub.
+            text: >
+                az iot hub message-endpoint list -n {iothub_name}
+          - name: Get all the endpoints of type "EventHub" from an IoT Hub.
+            text: >
+                az iot hub message-endpoint list -n {iothub_name} --endpoint-type eventhub
+    """
+
+    helps[
+        "iot hub message-endpoint show"
+    ] = """
+        type: command
+        short-summary: Get information on mentioned endpoint for an IoT Hub.
+        examples:
+          - name: Get an endpoint information from an IoT Hub.
+            text: |
+                az iot hub message-endpoint show -n {iothub_name} --endpoint-name {endpoint_name}
+    """
+
+    helps[
+        "iot hub message-endpoint delete"
+    ] = """
+        type: command
+        short-summary: Delete all or mentioned endpoint for an IoT Hub.
+        long-summary: Delete an endpoint for an IoT Hub. We recommend that you delete any routes to the endpoint, before deleting the endpoint.
```

Comment:
How about automating the route deletion for users. That is, if a user deletes an endpoint, we prompt the user to delete all the associated routes. Once they confirm, we delete the routes followed by the endpoint.

Reply:
will put that as a backlog, this pr is about getting the new routes and endpoints out (with cosmos db)

Reply:
Sounds good